backend/langate/network/apps.py

```diff
@@ -65,3 +64,26 @@ def ready(self):
                     mark_res = netcontrol.query("set_mark", {"mac": dev.mac, "mark": 100})
                     if not mark_res["success"]:
                         logger.info("[PortalConfig] Could not set mark 0 for device %s", dev.name)
+
+            logger.info(_("[PortalConfig] Add default whitelist devices to the ipset"))
+            if os.path.exists("assets/misc/whitelist.txt"):
+                with open("assets/misc/whitelist.txt", "r") as f:
+                    for line in f:
+                        line = line.strip().split("|")
+                        if len(line) == 2:
```

Comment:
Du coup on peut pas configurer la mark des appareils whitelist ?

Reply:
Everything related to Marks was delayed to #23 which is still in progress.